pkg/db/query/job_queries.go

```diff
@@ -12,6 +12,56 @@ import (
 	"github.com/openshift/sippy/pkg/filter"
 )
 
+func JobRunTestCount(dbc *db.DB, jobRunID int64) (int, error) {
+	var prowJobRunTestCount int
+	var tests []models.ProwJobRunTest
+
+	res := dbc.DB.Find(&tests, "prow_job_run_id = ?", jobRunID)
+
+	if res.Error != nil {
+		return -1, res.Error
+	}
+
+	prowJobRunTestCount = len(tests)
+
+	return prowJobRunTestCount, nil
+}
+
+func ProwJobHistoricalTestCounts(dbc *db.DB, prowJobID uint) (int, error) {
+
+	var jobRuns []models.ProwJobRun
+	res := dbc.DB.Joins("ProwJob").
+		Preload("Tests").Order("updated_at desc").Limit(10).Find(&jobRuns, "prow_job_id = ? AND succeeded = true", prowJobID)
```

Comment:
Nit: Suggest using "timestamp" instead of updated_at, which is the time the job ran rather than when we went into the db.

Should we filter here on timestamp > now - 2 weeks? Wondering if there are concerns with finding those successes like 3 months back.